app/models/user.rb

```diff
@@ -79,6 +97,10 @@ def update_mailchimp
     Rollbar.error(e)
   end
 
+  def confirmation_period_expired?
+    super
+  end
```

Comment:
this seems like it doesn't need to be here

Reply:
So that's what I thought but annoyingly you can't call the method from outside the class without this being there

Reply:
ah right, I guess we're using an undocumented and private method from the underlying library, so if we do it this way or the way previously, it's a bit naughty. So I'm happy either way.